api/middleware.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const helmet = require("helmet");
+const morgan = require("morgan");
+const cors = require("cors");
+
+module.exports = server => {
+  server.use(express.json());
+  server.use(helmet());
+  server.use(morgan("dev"));
+  server.use(cors());
+};
```

Comment:
nice, I like how you have all middleware separate from your main server. solid reasoning based on separation of concerns